src/Aspect/HttpClientMetricAspect.php

```diff
@@ -74,8 +77,15 @@ private function onFullFilled(Timer $timer, array $labels): callable
 
     private function onRejected(Timer $timer, array $labels): callable
     {
-        return function (RequestException $exception) use ($timer, $labels) {
-            $labels['http_status_code'] = (string) $exception->getResponse()->getStatusCode();
+        return function (TransferException $exception) use ($timer, $labels) {
+            $labels['http_status_code'] = '';
+            $labels['message'] = Str::snake($exception->getMessage());
```

Comment:
O message já foi removido num incidente há um tempo atrás que estourou a cardinalidade das métricas. Se for tratado só para os casos específicos, vai ser uma boa.

[#sanitização-cardinalidade-metricas](https://picpay.slack.com/archives/C05SJTJ40Q1)

https://github.com/opencodeco/hyperf-metric/pull/2/files (editado)

Reply:
Adicionei mensagens específicas que acredito ser o suficiente para acompanhar os dois tipos de erros.